cms/appresolver.py

```diff
@@ -23,12 +23,10 @@ def clear_app_resolvers():
     APP_RESOLVERS = []
 
 
-def applications_page_check(request, current_page=None, path=None):
+def applications_page_check(request, path=None):
     """Tries to find if given path was resolved over application.
     Applications have higher priority than other cms pages.
     """
-    if current_page:
```

Comment:
@marksweb How do you know that this is unused? This could be required for apphooks.

Reply:
@Aiky30 Based on the discussions on the associated issue (#3378) and a search of the project for use of this function.

The original issue is old, but has input from core members who agree that it's superfluous and likely stems from before the switch to treebeard.